src/user/dto/list-user-by-id.dto.ts

```diff
@@ -0,0 +1,14 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { IsNotEmpty, IsUUID } from 'class-validator';
+
+export class GetUserDto {
+  @IsNotEmpty()
+  @IsUUID()
+  @ApiProperty({
+    example: '290b6435-13fc-4f12-8550-522ed28134fb',
+    description: 'User id to validate get request',
```

Comment:
Pode deixar em português a descrição do Swagger sem problemas, e não precisa colocar "to validade get request"

"Id do usuário."

Reply:
Arrumado!